api/jobset/v1alpha2/jobset_types.go

```diff
@@ -24,6 +24,7 @@ const (
 	ReplicatedJobReplicas string = "jobset.sigs.k8s.io/replicatedjob-replicas"
 	ReplicatedJobNameKey  string = "jobset.sigs.k8s.io/replicatedjob-name"
 	JobIndexKey           string = "jobset.sigs.k8s.io/job-index"
+	JobHashKey            string = "jobset.sigs.k8s.io/job-hash"
```

Comment:
I would call it `jobset.sigs.k8s.io/job-key`, hash is too specific.

Reply:
done